Assets/CritterController.cs

```diff
@@ -0,0 +1,135 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using System.Linq;
+
+public class CritterController : MonoBehaviour
+{
+    public float consumed, hydrated;
+    public float consumptionLostPerSecond;
+    public float hydrationLostPerSecond;
+    public float force;
+
+    public GameObject target;
+    private Rigidbody rb;
+
+    void Start()
+    {
+        gameObject.GetComponent<Renderer>().material.color = Color.red;
+
+        rb = gameObject.GetComponent<Rigidbody>();
+    }
+
+    void Update()
+    {
+        consumed -= consumptionLostPerSecond * Time.deltaTime;
+        hydrated -= hydrationLostPerSecond * Time.deltaTime;
+
+        if (consumed <= 0 || hydrated <= 0)
+        {
+            Die();
+        }
+
+        target = ChooseTarget();
+
+        if (target)
+        {
+            // accelerate in the direction of the target
+            // (This is not a very good aiming strategy.)
+            Vector3 forceToMove = target.transform.position - transform.position;
+            forceToMove.y = 0;
+            forceToMove.Normalize();
```

Comment:
After normalizing we should be multiplying by some constant, being deliberate about how strong we decide the creature's acceleration is.

Reply:
I think line 43 does that. `force` is a constant that can be edited in the Unity GUI.

Reply:
Roger that! I totally missed it.